### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by yangfl**
This change is [Reviewable](https://reviewable.io/reviews/utox/utox/1374)

### CODE REVIEW

File: src/android/audio.c [hunk 1]
```diff
@@ -29,7 +29,7 @@ static SLObjectItf                   recorderObject = NULL;
 static SLRecordItf                   recorderRecord;
 static SLAndroidSimpleBufferQueueItf recorderBufferQueue;
 
-// dont change this
+// don't change this
```
Comment (GrayHatter) [inline src/android/audio.c]: delete this comment instead

File: src/friend.c [hunk 2]
```diff
@@ -27,7 +27,7 @@
 static FRIEND *friend = NULL;
 
 FRIEND *get_friend(uint32_t friend_number) {
-    if (friend_number >= self.friend_list_size) { //friend doesnt exist if true
+    if (friend_number >= self.friend_list_size) { //friend doesn't exist if true
```
Comment (GrayHatter) [inline src/friend.c]: delete this comment instead

File: src/friend.c [hunk 3]
```diff
@@ -58,7 +58,7 @@ static FREQUEST *frequests = NULL;
 static uint16_t frequest_list_size = 0;
 
 FREQUEST *get_frequest(uint16_t frequest_number) {
-    if (frequest_number >= frequest_list_size) { //frequest doesnt exist if true
+    if (frequest_number >= frequest_list_size) { //frequest doesn't exist if true
```
Comment (GrayHatter) [inline src/friend.c]: delete this comment instead

File: src/android/main.c [hunk 4]
```diff
@@ -169,7 +169,7 @@ FILE *native_get_file_simple(const char *path, UTOX_FILE_OPTS opts) {
 
     if (!fp && opts & UTOX_FILE_OPTS_READ && opts & UTOX_FILE_OPTS_WRITE) {
         LOG_WARN("Android Native", "Unable to simple open, falling back to fd" );
-        // read wont create a file if it doesn't' already exist. If we're allowed to write, lets try
+        // read won't create a file if it doesn't' already exist. If we're allowed to write, lets try
```
Comment (avoidr) [inline src/android/main.c]: "doesn't'" -> "doesn't"
"lets" -> "let's"

File: src/avatar.h [hunk 5]
```diff
@@ -103,7 +103,7 @@ bool avatar_save(char hexid[TOX_PUBLIC_KEY_SIZE * 2], const uint8_t *data, size_
  */
 bool avatar_delete(char hexid[TOX_PUBLIC_KEY_SIZE * 2]);
 
-/* Helper function to intialize the users avatar */
+/* Helper function to initialize the users avatar */
```
Comment (avoidr) [inline src/avatar.h]: "users" -> "user's"

File: src/messages.c [hunk 6]
```diff
@@ -965,10 +965,10 @@ static void messages_draw_filetransfer(MESSAGES *m, MSG_FILE *file, uint32_t i,
     drawtextrange(dx + SCALE(10), wbound - SCALE(10), y + SCALE(6), ft_text, ft_text_length);
 }
 
-/* This is a bit hacky, and likely would benifit from being moved to a whole new section including seperating
+/* This is a bit hacky, and likely would benefit from being moved to a whole new section including separating
  * group messages/functions from friend messages and functions from inside ui.c.
  *
- * Idealy group and friend messages wouldn't even need to know about eachother.   */
+ * Idealy group and friend messages wouldn't even need to know about each other.   */
```
Comment (avoidr) [inline src/messages.c]: "Idealy" -> "Ideally"

File: src/posix/filesys.c [hunk 7]
```diff
@@ -105,7 +105,7 @@ FILE *native_get_file_simple(const char *path, UTOX_FILE_OPTS opts) {
 
     if (!fp && opts & UTOX_FILE_OPTS_READ && opts & UTOX_FILE_OPTS_WRITE) {
         LOG_WARN("POSIX", "Unable to simple open, falling back to fd" );
-        // read wont create a file if it doesn't' already exist. If we're allowed to write, lets try
+        // read won't create a file if it doesn't' already exist. If we're allowed to write, lets try
```
Comment (avoidr) [inline src/posix/filesys.c]: "doesn't'" -> "doesn't"
"lets" -> "let's"

File: src/tox.c [hunk 8]
```diff
@@ -583,7 +583,7 @@ void toxcore_thread(void *UNUSED(args)) {
  * There are two main threads, the tox worker thread, that interacts with Toxcore, and receives the callbacks. The other
  * is the 'uTox' thread that interacts with the user, (rather sends information to the GUI.) The tox thread and the uTox
  * thread may interact with each other, as you see fit. However the Toxcore thread has child threads that are a bit
- * temperamental. The ToxAV thread is a child of the Toxcore thread, and therefor will ideally only be called by the tox
+ * temperamental. The ToxAV thread is a child of the Toxcore thread, and therefore will ideally only be called by the tox
  * thread. The ToxAV thread also has two children of it's own, an audio and a video thread. Both a & v threads should
```
Comment (avoidr) [inline src/tox.c]: "it's" -> "its"

File: src/tox_callbacks.c [hunk 9]
```diff
@@ -219,12 +219,12 @@ static void callback_group_peer_name_change(Tox *UNUSED(tox), uint32_t gid, uint
 
     if (g->peer) {
         if (!g->peer[pid]) {
-            LOG_ERR("Tox Callbacks", "Tox Group:\tERROR, can't sent a name, for non-existant peer!" );
+            LOG_ERR("Tox Callbacks", "Tox Group:\tERROR, can't sent a name, for non-existent peer!" );
```
Comment (avoidr) [inline src/tox_callbacks.c]: "sent" -> "set"

File: src/tox_callbacks.c [hunk 10]
```diff
@@ -219,12 +219,12 @@ static void callback_group_peer_name_change(Tox *UNUSED(tox), uint32_t gid, uint
...
             return;
         }
     } else {
         // TODO can't happen
-        LOG_ERR("Tox Callbacks", "Tox Group:\tERROR, can't sent a name, for non-existant Group!" );
+        LOG_ERR("Tox Callbacks", "Tox Group:\tERROR, can't sent a name, for non-existent Group!" );
```
Comment (avoidr) [inline src/tox_callbacks.c]: "sent" -> "set"

File: src/tox_callbacks.c [hunk 11]
```diff
@@ -303,7 +303,7 @@ static void callback_group_topic(Tox *UNUSED(tox), uint32_t gid, uint32_t pid, c
 void callback_group_connected(Tox *UNUSED(tox), uint32_t gid, void *UNUSED(userdata)){
     GROUPCHAT *g = get_group(gid);
     if (!g) {
-        LOG_ERR("Tox Callbacks", "Toxcore says were connected to a non-existant groupchat %u.", gid);
+        LOG_ERR("Tox Callbacks", "Toxcore says were connected to a non-existent groupchat %u.", gid);
```
Comment (avoidr) [inline src/tox_callbacks.c]: "were" -> "we're"

**Comment by avoidr** [comment]
Merged, thanks!

**Pull request merged**